tests/e2e/ansible/start_docker_registry.yaml

```diff
@@ -7,12 +7,12 @@
 #
 ---
 - hosts: all
-  become: yes
   vars:
     local_registry_port: 5000
     local_registry_name: local-registry
   tasks:
     - name: Install pip3
+      become: yes
```

Comment:
On my kcli ubuntu I need a couple of more sudos to make this passing, not sure how my setup is different

```
diff --git a/tests/e2e/ansible/start_docker_registry.yaml b/tests/e2e/ansible/start_docker_registry.yaml
index 213fc47..dd50193 100644
--- a/tests/e2e/ansible/start_docker_registry.yaml
+++ b/tests/e2e/ansible/start_docker_registry.yaml
@@ -31,6 +31,7 @@
       retries: 3
       delay: 10
     - name: Start a docker registry
+      become: yes
       docker_container:
         name: "{{ local_registry_name }}"
         # TODO: docker.io is troublesome as we can hit the pull limits.
@@ -44,6 +45,7 @@
     # Remove the docker registry container.
     #
     - name: Remove the docker registry container
+      become: yes
       tags: undo
       block:
         - name: Remove the docker registry
```

Reply:
That's needed because your user is not properly configured / added to the docker group.

Reply:
Nevermind, I removed this part as I could get rid of using pip in general (I think).